bluepysnap/edges/edge_population.py

```diff
@@ -627,6 +627,12 @@ def spatial_synapse_index(self):
             raise BluepySnapError(f"It appears {self.name} does not have synapse indices")
         return open_index(index_dir)
 
+    # ALIASES
+    edges_by_source = efferent_edges
+    edges_by_target = afferent_edges
+    target_nodes_by_source = efferent_nodes
+    source_nodes_by_target = afferent_nodes
```

Comment:
I find the naming a bit clunky. `outgoing_edges` and `incoming_edges` would be IMO more intuitive for eff and aff edges.

`target_nodes_by_source` also sounds very confusing!

Reply:
> I find the naming a bit clunky. `outgoing_edges` and `incoming_edges` would be IMO more intuitive for eff and aff edges.

Sure, whatever makes sense to everybody is fine by me.

> `target_nodes_by_source` also sounds very confusing!

What would you suggest? I can always elaborate it with `get_target_nodes_of_source_nodes` or whatever. I just want something that people will immediately understand so they don't accidentally choose the incorrect function for their purposes.

Reply:
What about just `target_nodes` and `source_nodes`?
They would match the names of the underlying libsonata edges population [target_nodes](https://libsonata.readthedocs.io/en/stable/api.html#libsonata.EdgePopulation.target_nodes) and [source_nodes](https://libsonata.readthedocs.io/en/stable/api.html#libsonata.EdgePopulation.source_nodes)

Reply:
In libsonata, they expect edge ids as input (as opposed to node ids), right? Wouldn't that be even more confusing?